### PR TITLE
ENH v1.15.1

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #30

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
This PR supersedes #30.

@jakirkham for viz and to press the merge button if it works...

**Comment by beckermr** [comment]
This one looks good to go!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% set version = "1.15.1" %}
 
 package:
   name: blosc
   version: {{ version }}
 
 source:
-  url: https://github.com/Blosc/c-blosc/archive/v{{ version }}.tar.gz
-  sha256: 42a85de871d142cdc89b607a90cceeb0eab60d995f6fae8d44aae397ab414002
+  url: https://github.com/Blosc/c-blosc/archive/{{ version }}.tar.gz
```
Comment (beckermr) [inline recipe/meta.yaml]: The patched version doesn't have a `v` in the tag. :(

**Comment by beckermr** [comment]
OK @jakirkham. This PR is ready to merge. The patched version was released, so I removed the patch here and went ahead with the update.

**Comment by beckermr** [comment]
Also @conda-forge/blosc feel free to press the merge button. I know @jakirkham is quite busy.

**Comment by jakirkham** [comment]
Thanks @beckermr 😄

**Pull request merged**